aptos-move/framework/move-stdlib/src/natives/bcs.rs

```diff
@@ -126,6 +134,38 @@ fn serialized_size_impl(
     serialized_size_allowing_delayed_values(&value, &ty_layout)
 }
 
+fn native_constant_serialized_size(
+    context: &mut SafeNativeContext,
+    mut ty_args: Vec<Type>,
+    _args: VecDeque<Value>,
+) -> SafeNativeResult<SmallVec<[Value; 1]>> {
+    debug_assert!(ty_args.len() == 1);
+
+    context.charge(BCS_CONSTANT_SERIALIZED_SIZE_BASE)?;
+
+    let ty = ty_args.pop().unwrap();
```

Comment:
nit: I think Victor added some "safe" pops, `safely_pop_arg` or something like that.

Reply:
this is type not arg. 

rest of the functions in this file have this same line